chaoscenter/authentication/api/handlers/rest/project_handler.go

```diff
@@ -55,14 +55,17 @@ func GetUserWithProject(service services.ApplicationService) gin.HandlerFunc {
 
 		outputUser := user.GetUserWithProject()
 
-		projects, err := service.GetProjectsByUserID(outputUser.ID, false)
+		request := utils.GetProjectFilters(c)
+		request.UserID = outputUser.ID
+
+		response, err := service.GetProjectsByUserID(request)
```

Comment:
Can we remove the user.GetUserWithProject() func call completely and use just the above function? Either we can get userID from getUser() or tweak the above function to search by username as well

Reply:
sure we remove user.GetUserWithProject and user GetUser and GetProjectsByUserID

Reply:
we should do this other pr because this is little complex the function is also being used in the frontend.

Reply:
done